internal/analytics/analytics.go

```diff
@@ -125,7 +128,7 @@ func generateEventName(command string, action string) string {
 	commands := strings.Split(command, " ")
 
 	for i := range commands {
-		commands[i] = strings.Title(commands[i])
+		commands[i] = cases.Title(language.English).String(commands[i])
```

Comment:
Fixes lint error:

```
Error: SA1019: strings.Title has been deprecated since Go 1.18 and an alternative has been available since Go 1.0: The rule Title uses for word boundaries does not handle Unicode punctuation properly. Use golang.org/x/text/cases instead.  (staticcheck)
 ```